xclim/core/missing.py

```diff
@@ -56,13 +58,15 @@ class MissingBase:
     to the registry before using them in an Indicator.
     """
 
-    def __init__(self, da, freq, **indexer):
-        self.null, self.count = self.prepare(da, freq, **indexer)
+    def __init__(self, da, freq, input_freq, **indexer):
+        if input_freq is None:
+            raise ValueError("`input_freq` must be either 'D' or 'H'.")
+        self.null, self.count = self.prepare(da, freq, input_freq, **indexer)
 
     @classmethod
-    def execute(cls, da, freq, options, indexer):
+    def execute(cls, da, freq, input_freq, options, indexer):
```

Comment:
I think I prefer `src_timestep` in the end

Reply:
Ok, will make the change and merge.